product-schema.json

```diff
@@ -71,202 +71,22 @@
-                "$ref": "#/$defs/regex"
-              }
-            },
-            "required": [
-              "github_releases"
-            ],
-            "additionalProperties": false
-          },
-          {
-            "type": "object",
-            "properties": {
-              "pypi": {
-                "title": "Pypi",
-                "description": "Package name on pypi.",
-                "type": "string"
-              },
-              "regex": {
-                "$ref": "#/$defs/regex"
-              },
-              "regex_exclude": {
-                "$ref": "#/$defs/regex"
-              },
-              "template": {
-                "$ref": "#/$defs/template"
-              }
-            },
-            "required": [
-              "pypi"
-            ],
-            "additionalProperties": false
-          },
-          {
-            "type": "object",
-            "properties": {
-              "maven": {
-                "title": "Maven",
-                "description": "Package name on maven.",
-                "type": "string"
-              },
-              "regex": {
-                "$ref": "#/$defs/regex"
-              },
-              "regex_exclude": {
-                "$ref": "#/$defs/regex"
-              },
-              "template": {
-                "$ref": "#/$defs/template"
-              }
-            },
-            "required": [
-              "maven"
-            ],
-            "additionalProperties": false
-          },
-          {
-            "type": "object",
-            "properties": {
-              "cgit": {
-                "title": "Cgit",
-                "description": "Fetch versions (from tags) with their dates from a cgit repository.",
-                "type": "string"
-              },
-              "regex": {
-                "$ref": "#/$defs/regex"
-              },
-              "regex_exclude": {
-                "$ref": "#/$defs/regex"
-              },
-              "template": {
-                "$ref": "#/$defs/template"
-              }
-            },
-            "required": [
-              "cgit"
-            ],
-            "additionalProperties": false
+      "type": "object",
+      "required": ["methods"],
+      "properties": {
+        "cumulative": {
+          "title": "Mark auto-update as being cumulative (optional, default = false)",
+          "description": "When true, data won't be deleted before fetching new data.\nActivating cumulative updates is not recommended for most products, but could be useful for products that:\n - have a long history of releases that is long to fetch,\n - have a history of releases that is not available anymore.",
+          "type": "boolean"
+        },
+        "methods": {
+          "title": "Auto-update configs",
+          "description": "See https://github.com/endoflife-date/endoflife.date/wiki/Automation",
+          "type": "array",
+          "items": {
+            "$ref": "#/$defs/automethod"
```

Comment:
commit 4961fb8e189ba252a3e6262522e88e7458dc9020 :eyes: 

repairs product-schema.json staleness introduced in 1312fab559fa13d4c4950b816f7b9747a8601671 — the actual schema has changed `[method1, method2]` → `{"cumulative": bool, "methods": [method1, method2]}`.

the large `oneOf` switch was factored out into `$defs` below.